test/functional/test-url-replacements.js

```diff
@@ -185,6 +185,18 @@ describes.sandboxed('UrlReplacements', {}, () => {
     return win;
   }
 
+  it('limit replacement params size', () => {
+    return getReplacements().then(replacements => {
+      replacements.getVariableSource().initialize();
+      const variables =
+          Object.keys(replacements.getVariableSource().replacements_);
+      // Restrict the number of replacement params to globalVaraibleSource
+      // Please consider adding the logic to amp-analytics instead.
+      // Please contact @lannka or @zhouyx if the test fail.
+      expect(variables).to.have.lengthOf.within(0, 69);
```

Comment:
let's make it exactly equal.
so we timely reduce the size when we move things out.

Reply:
Good point. Done